<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="D11-KamalKant.local" name="tests.commonAPI.CommonAPITests" tests="48" failures="10" timestamp="01 Oct 2020 06:03:41 GMT" time="42.785" errors="0">
  <testcase name="getCityWithName" time="0.847" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.839" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCitiesWithInvalidLocation" time="0.937" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.commonAPI.CommonAPITests.getCitiesWithInvalidLocation(CommonAPITests.java:174)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCitiesWithInvalidLocation -->
  <testcase name="getCityWithDiffParam" time="0.695" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdAndName" time="0.746" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.881" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.786" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.724" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidCityId" time="0.808" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCollection(Common.java:60)
at tests.commonAPI.CommonAPITests.getCollectionWithInvalidCityId(CommonAPITests.java:181)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCollectionWithInvalidCityId -->
  <testcase name="getCityWithLocation" time="0.955" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.977" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.984" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.commonAPI.CommonAPITests.getCityWithDiffParam(CommonAPITests.java:77)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffParam -->
  <testcase name="getCityWithIdAndName" time="0.976" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.856" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidLocation" time="0.833" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCollection(Common.java:60)
at tests.commonAPI.CommonAPITests.getCollectionWithInvalidLocation(CommonAPITests.java:166)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCollectionWithInvalidLocation -->
  <testcase name="getCityWithDiffCount" time="1.047" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.commonAPI.CommonAPITests.getCityWithDiffCount(CommonAPITests.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffCount -->
  <testcase name="getCityWithDiffParam" time="0.939" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="1.086" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithInvalidQParam" time="0.784" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.commonAPI.CommonAPITests.getCityWithInvalidQParam(CommonAPITests.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidQParam -->
  <testcase name="getCityWithDiffParam" time="0.745" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdAndName" time="1.177" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCategoriesTest" time="1.098" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.847" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.869" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffCount" time="0.823" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.723" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.813" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [3]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.commonAPI.CommonAPITests.getCityWithDiffParam(CommonAPITests.java:77)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffParam -->
  <testcase name="getCityWithInvalidQParam" time="0.890" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.commonAPI.CommonAPITests.getCityWithInvalidQParam(CommonAPITests.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidQParam -->
  <testcase name="getCityWithName" time="0.803" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.886" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidQParam" time="0.957" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidQParam" time="0.732" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.920" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdAndName" time="0.909" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.939" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.795" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.855" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.780" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithName" time="0.823" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.952" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithInvalidIdNameCombination" time="0.875" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.commonAPI.CommonAPITests.getCityWithInvalidIdNameCombination(CommonAPITests.java:134)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidIdNameCombination -->
  <testcase name="getCityWithName" time="0.996" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffCount" time="1.134" classname="tests.commonAPI.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [10]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.commonAPI.CommonAPITests.getCityWithDiffCount(CommonAPITests.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffCount -->
  <testcase name="getCityWithIdOnly" time="0.817" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="1.062" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="1.068" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="0.908" classname="tests.commonAPI.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="0.889" classname="tests.commonAPI.CommonAPITests"/>
</testsuite> <!-- tests.commonAPI.CommonAPITests -->
